eth/filters/filter_system.go

```diff
@@ -52,6 +52,8 @@ const (
 	PendingTransactionsSubscription
 	// BlocksSubscription queries hashes for blocks that are imported
 	BlocksSubscription
+	// BlockResultSubscription queries evmTrace list when new block created
+	BlockResultSubscription
```

Comment:
`BlockResultsSubscription `

Reply:
The basic unit is the execution result of a single block, not a batch, so `blockResult` maybe suitable do you think?